docs/tutorials/studio-tutorial.md

```diff
@@ -572,6 +572,15 @@ This panel lets you define tests for the services you write by specifying test d
 
 ## Create a Service Test 
 
+### Create connections test data
```

Comment:
I think we should move the Test Data to its own section in the docs. we can reference it in the service test. But they will be shared along for mapping tests as well. Please reorganize

Reply:
+1
Test Data will be used across the platform with multiple diff elements